public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -4,20 +4,40 @@ import React from 'react';
 import { GrafanaTheme2, colorManipulator } from '@grafana/data';
 import { useStyles2 } from '@grafana/ui';
 
+export interface PublicDashboardFooter {
+  disable: boolean;
+  text: string;
+  logo: string;
+  link: string;
+}
+
 export const PubdashFooter = function () {
```

Comment:
I think we should use PublicDashboard naming. it's everywhere like this

Reply:
https://github.com/grafana/grafana-partnerships-team/issues/480 -> I added this chore issue to track that, as is not part of this task